Brokerages/Bitfinex/BitfinexBrokerage.cs

```diff
@@ -175,10 +201,10 @@ public override List<Order> GetOpenOrders()
                     continue;
                 }
 
-                order.Quantity = item.Side == "sell" ? -item.OriginalAmount : item.OriginalAmount;
-                order.BrokerId = new List<string> { item.Id };
+                order.Quantity = item.Amount;
+                order.BrokerId = new List<string> { item.Id.ToStringInvariant() };
                 order.Symbol = _symbolMapper.GetLeanSymbol(item.Symbol);
-                order.Time = Time.UnixTimeStampToDateTime(item.Timestamp);
+                order.Time = Time.UnixMillisecondTimeStampToDateTime(item.MtsCreate);
```

Comment:
We might be able to set the LastUpdateTime with `MtsUpdate` I think, maybe if different from `MtsCreate`?

Reply:
I'll check if `MtsUpdate` is also updated on each fill, in this case we should update `LastFillTime` instead